section-engineering-education/engineering-education
# Total likes insights

- Refactor how we fetch data from Hyvor from multiple fetch calls to one.
- Calculate all likes from each article and display on the top insights.

**How to Test:**
Navigate to beta.section.io/engineering-education/ and visit a students profile page (or checkout the code locally). The insights section will show up, but there wont be any data available for likes due to the staging url. If you pull locally and test there you can visit this page to see likes data: http://localhost:1313/authors/grace-mumbi/

The design should match the screenshot and should have a hover tooltip describing the insight.
<img width="1087" alt="image" src="https://user-images.githubusercontent.com/15935329/163444086-82c36db6-0b40-4f32-bbd5-82b157a21faa.png">
